packages/app-builder/src/components/Cases/Filters/CasesFiltersContext.tsx

```diff
@@ -29,6 +30,7 @@ export type CasesFiltersForm = {
   statuses: CaseStatus[];
   dateRange: DateRangeFilterForm;
   name?: string;
+  snoozed?: 'true' | 'false';
```

Comment:
Suggestion : Following the previous comment.
```suggestion
  snoozed?: boolean;
```

Reply:
Responded on my first answer :)